Userland/Applications/Spreadsheet/Spreadsheet.cpp

```diff
@@ -535,9 +540,7 @@ JsonObject Sheet::to_json() const
         JsonObject metadata_object;
         metadata_object.set("length", meta.length);
         metadata_object.set("format", meta.format);
-#if 0
-        metadata_object.set("alignment", alignment_to_string(meta.alignment));
-#endif
+        metadata_object.set("alignment", to_string(meta.alignment));
```

Comment:
I'm not sure why this is pulled into the global namespace, but please qualify it with `Gfx::`.

Reply:
Done